torchvision/models/detection/rpn.py

```diff
@@ -300,8 +313,17 @@ def _get_top_n_idx(self, objectness, num_anchors_per_level):
         r = []
         offset = 0
         for ob in objectness.split(num_anchors_per_level, 1):
-            num_anchors = ob.shape[1]
-            pre_nms_top_n = min(self.pre_nms_top_n, num_anchors)
+            if torchvision._is_tracing():
+                from torch.onnx import operators
```

Comment:
Ok, let's keep this implementation as is for now. But could you move this code into a separate function, like
```python
@torch.jit.unused
def _onnx_get_num_anchors_and_pre_nms_top_n(ob, orig_pre_nms_top_n):
    # implementation goes here
```